app/assets/stylesheets/_related.scss

```diff
@@ -3,16 +3,16 @@
   -webkit-box-sizing: border-box;
   -moz-box-sizing: border-box;
   box-sizing: border-box;
-  margin-top: 40px
+  margin-top: 40px;
```

Comment:
😨  does this mean the `40px` was never applied but now is. do you know the impact if so?

Reply:
I have answered my own question on this

<img width="656" alt="Screen Shot 2019-11-11 at 17 22 02" src="https://user-images.githubusercontent.com/7016425/68607178-eff3c000-04a7-11ea-86d3-3b32f6c2f056.png">